JS/Lesson19/gulcin/recursion.js

```diff
@@ -88,10 +88,38 @@ console.log(vowelsInString(sentenceEx4));
 // 4. Write a recursive function to remove all occurrences of a specified character from a string.
 const sentenceEx5 = 'Hello, how are you?';
 const charToRemove = 'o';
+function removeChar(sentence, charToDelete) {
+	if(sentence.length === 0) {
+		return '';
+	} else {
+		if(sentence[0] === charToDelete[0]) {
+          return removeChar(sentence.slice(1), charToDelete);
+		} else {
+		  return sentence[0] + removeChar(sentence.slice(1), charToDelete);
+		}
+	};
+}
+console.log(removeChar(sentenceEx5, charToRemove));
 
 // 5. Write a recursive function to check if an array includes a specific value.
 const numbers = [1, 2, 3, 4, 5];
-const valueToCheck = 3;
+const valueToCheck = 4;
+const valueToCheck2 = 10;
+
+function checkValInArr(arr, valToCheck) {
+	if(arr.length === 0) {
+		return `${valToCheck} is not found!`;
```

Comment:
Better to return false here as the function is expected to return either true or false
```suggestion
		return false;
```